client/server/staking.go

```diff
@@ -294,27 +294,29 @@ func (s *Server) GetDelegatorByDelegatorAddress(r *http.Request) (resp any, err
 
 	delAddr := mux.Vars(r)["delegator_addr"]
 
+	result := map[string]string{
```

Comment:
We should use a struct with a proper naming here. By default, string type is "".